app/models/course.rb

```diff
@@ -3,5 +3,20 @@ class Course < ApplicationRecord
   belongs_to :trimester
   has_many :enrollments
 
-  delegate :title, to: :coding_class
+   def student_name_list
+    names_list = []
+    self.enrollments.each do |enrollment|
+      names_list << "#{enrollment.student.first_name} #{enrollment.student.last_name}"
+    end
+    names_list
```

Comment:
```suggestion

    names_list
```
nit: Indentation before implicit return after a block